tox.ini

```diff
@@ -9,8 +9,7 @@ env_list = format, lint, unit
 [vars]
 src_path = {tox_root}/src
 tests_path = {tox_root}/tests
-lib_path = {tox_root}/lib
-all_path = {[vars]src_path} {[vars]tests_path} {[vars]lib_path}
```

Comment:
Linting fails for the lib, but it's not really the data-integrator's job to check it.